third_party/silabs/efr32_sdk.gni

```
@@ -49,10 +49,25 @@ declare_args() {
 
   # Argument to Disable IPv4 for wifi(rs911)
   chip_enable_wifi_ipv4 = false
+  silabs_gen_folder = ""
+
+  # Calls SLC-CLI at run time
+  generate_slc = false
+
+  # Use SLC generated files
+  use_slc_files = false
```

```suggestion
  slc_reuse = false
```
Asymmetry makes my brain squirm.